include/dfu/dfu_target.h

```diff
@@ -99,6 +100,17 @@ int dfu_target_offset_get(size_t *offset);
  **/
 int dfu_target_write(const void *const buf, size_t len);
 
+/**
+ * @brief Erase the DFU storage for the initialized DFU target.
+ *
+ * @param[in] force Indicate whether the erase should be exectued even if the
+ *		    area already reports as empty.
+ *
+ * @return 0 for an successful erase or a negative error
+ *	   code identicating reason of failure.
+ **/
+int dfu_target_erase(int slot, bool force);
```

Comment:
Missing description for `slot` parameter (for this and all the target functions).